subprojects/docs/src/docs/release/notes.md

```diff
@@ -272,6 +272,33 @@ There are better ways for re-using task logic, for example by using [task depend
 
 The default output location in [EclipseClasspath](dsl/org.gradle.plugins.ide.eclipse.model.EclipseClasspath.html#org.gradle.plugins.ide.eclipse.model.EclipseClasspath:defaultOutputDir) changed from `${project.projectDir}/bin` to `${project.projectDir}/bin/default`.
 
+### Blacklist repository on `IOException`s 
+
+In the event of a timeout (or any `IOException`s which are not caused by error HTTP status code), Gradle will skip subsequent connections to the same repository for the duration of the build. 
```

Comment:
⭕️ Let's just say "in case of repository failures". IOException is an implementation detail.

Reply:
Done.